src/Validation/Rules/Password.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Validation\Rules;
+
+use Attribute;
+use Tempest\Validation\Rule;
+
+#[Attribute]
+final readonly class Password implements Rule
+{
+    private int $min;
+    private ?int $max;
+
+    public function __construct(
+        int $min = 8,
+        ?int $max = null,
+        private bool $mixedCase = false,
+        private bool $numbers = false,
+        private bool $letters = false,
+        private bool $symbols = false,
+    ) {
```

Comment:
Let's add an option along the lines of `$restrictRepetitiveCharacters` (if you think of naming more user friendly, I'm open to it!)

Reply:
Can you define this in a little more detail?

Reply:
Actually, ignore this for now. We can revisit later if we need to.

Was thinking things like "aaaaaa"